src/main/resources/application.conf

```diff
@@ -1,7 +1,18 @@
-http {
+cromiam {
+  user_id_header = "X-User-Id"
+  allowed_users = [ "yossi" ]
```

Comment:
I'm assuming this is a placeholder but "yossi" is probably not the correct string :)